src/createAnimatedComponent.js

```diff
@@ -174,19 +175,30 @@ export default function createAnimatedComponent(Component) {
       }
     }
 
+    shouldComponentUpdate(nextProps) {
```

Comment:
We need to attach new props before re-render in order to handle new props.

Reply:
Tried to move in to render but it gives waning as it's not advisable to use `findNodeHandle` inside `render`. Suppose that it changes nothing but this warning is unpleasant